### ISSUE TITLE
Is this library being maintained?

### ISSUE THREAD

**Issue by Jazz747** [recr0ns/react-native-material-switch#27]
I see a lot of issues building up and PR's not merging. If this one isn't active, does anyone know any good alternatives?

**Comment by anti-nerd** [comment]
Was wondering the same thing. I've personally just opted to use the RN Switch component since I can't find any other alternative online (or one that I like for that matter). Better safe than sorry.

**Comment by Jazz747** [comment]
@anti-nerd I've switched to this library:

https://github.com/xinthink/react-native-material-kit

and its Switch component is working even better than this one ;)

**Comment by anti-nerd** [comment]
@Jazz747 just checked it out. Looks pretty neat. I'll surely try it out in one of my projects. Thanks, mate! :)

**Comment by duycoder** [comment]
Well, this library still can working. You go to your project and go to **~/node_modules/react-native-material-switch/index.js** and replace"**require**" by "**import**" and change the  standard library that I'm pretty sure that you already know. By the way, i think this library looks pretty good :D

**Comment by luck2011** [comment]
@duycoder No, it uses ```React.createClass``` which is too old for projects.

**Comment by duycoder** [comment]
So you replace it by modern React Native. I'm using it right now
